app/scripts/controllers/permissions/enums.js

```diff
@@ -1,28 +1,15 @@
-export const APPROVAL_TYPE = 'wallet_requestPermissions';
-
 export const WALLET_PREFIX = 'wallet_';
 
-export const HISTORY_STORE_KEY = 'permissionsHistory';
-
-export const LOG_STORE_KEY = 'permissionsLog';
-
-export const METADATA_STORE_KEY = 'domainMetadata';
-
-export const METADATA_CACHE_MAX_SIZE = 100;
-
-export const CAVEAT_TYPES = {
-  limitResponseLength: 'limitResponseLength',
-  filterResponse: 'filterResponse',
-};
-
 export const NOTIFICATION_NAMES = {
   accountsChanged: 'metamask_accountsChanged',
   unlockStateChanged: 'metamask_unlockStateChanged',
   chainChanged: 'metamask_chainChanged',
 };
 
 export const LOG_IGNORE_METHODS = [
+  'wallet_addEthereumChain',
```

Comment:
Why are these now ignored?

Reply:
IMO they should always have been ignored, since they're not strictly related to permissions. I'm open to keeping them, though.

Reply:
Hmm. I wasn't aware that this log was meant just for permission-related calls. In any case, it's probably better to at least leave this as-is for this PR, to minimize the changes.

Reply:
Fair enough! b65e88909